src/personio_py/models.py

```diff
@@ -652,13 +668,49 @@ def to_dict(self, nested=False) -> Dict[str, Any]:
         return d
 
     def _create(self, client: 'Personio'):
-        pass
+        self._client.create_attendances([self])
```

Comment:
I left this part unfinished... `client` should have been an optional parameter and the function should choose between `client` and `self._client` or raise an error if both are None. I should probably fix this myself ^^